library/cwm/src/lib/cwm/table.rb

```diff
@@ -119,9 +166,16 @@ def multiselection?
   private
 
     def format_items(items)
-      items.map do |item|
-        Item(Id(item.first), *item[1..-1])
-      end
+      items.map { |i| format_item(i) }
+    end
+
+    # @see #format_items
+    def format_item(item)
+      return Item(Id(item.first), *item[1..-1]) if item.is_a?(Array)
+
+      children = format_items(item.children)
+      open_tag = item.open ? :open : :closed
```

Comment:
open/closed is a tricky one, but looks correct. I need to google it :) https://ell.stackexchange.com/questions/15373/close-vs-closed-and-open-vs-opened

Reply:
It's not tricky at all; it's very simple English: "The door is open" vs. "The door is closed".

It might be a bit confusing that they are using a past participle ("closed" from "to close") as an adjective in one case and a plain adjective ("open", not "opened") in another. But languages are not always logical or consistent.